Assets/Scripts/UserInterface/GameUI/LevelUpRewardScreenController.cs

```diff
@@ -0,0 +1,27 @@
+﻿using UnityEngine;
+using System.Collections;
+
+public class LevelUpRewardScreenController : UIController {
```

Comment:
The canvas appears to be blocking raycasts to the build button. Can you add a CanvasGroup component to it and uncheck "BlocksRaycasts?"